### DISCUSSION TITLE
Please create a big hint for `<!--more-->`

### DISCUSSION THREAD

**Discussion by user9209** [thegeeklab/hugo-geekblog#621] [Ideas]
Hi,

I am not sure, if there is the hint about `<!--more-->`.
**For me as a newcomer to greekblog it is one of the most important commands of geekblog!**

Thanks a lot.

**Question**: Can I search in the documentation? I did not find a simple search bar to enter `<!--more-->`.

**Comment by xoxys** [comment]
Hello, 

Not sure if I understand your request. This is just a theme for the static side generator, Hugo. As such, `<!--more-->` is a Hugo feature and unrelated to themes. The documentation can be found here https://gohugo.io/content-management/summaries/

**Reply by user9209** [reply]
Thanks, I am also newcomer for hugo ;-) Would be nice hint as a welcome to GeekBlog to avoid to long preview.
That's the answer to my question / request.

Thank you a lot for the themes GreekBlog and GreekDoc, very good!
